config/config.yaml

```diff
@@ -8,7 +8,7 @@ plank:
     timeout: 7200000000000 # 2h
     grace_period: 15000000000 # 15s
     utility_images:
-      clonerefs: "gcr.io/k8s-prow/clonerefs:v20190301-2d35634"
+      clonerefs: "quay.io/pusher/clonerefs:latest"
```

Comment:
We should pin this at some point, but it should be done after this PR is merged to master (and we have an automated build)